_literate/EX-airfoil/tutorial.jl

```diff
@@ -24,8 +24,6 @@ using PrettyPrinting
 import DataFrames
 import Statistics
 using CSV
-using PyPlot
```

Comment:
You haven't added a `using Plots` and you do have  `plot` call below. I'm guessing you just forgot. But more to the point is how this slipped by without you realising it. Or maybe I'm just missing something?

Reply:
Quite surprised because my workflow includes running the whole file and comparing any plots I fix to the version on the website. I have just went back to add `Plots` and `using Plots` and it worked as expected.

The only thing I can imagine is that I ran the file after activating the environment from a previous file when I had also run `using Plots` and without activating the environment of that file but this doesn't make sense because I removed `PyPlot` so I'm not sure.